testsupport/init.go

```diff
@@ -209,6 +209,8 @@ func WaitForDeployments(t *testing.T) wait.Awaitilities {
 		// Also verify the autoscaling buffer in both members
 		webhookImage := initMemberAwait.GetContainerEnv(t, "MEMBER_OPERATOR_WEBHOOK_IMAGE")
 		require.NotEmpty(t, webhookImage, "The value of the env var MEMBER_OPERATOR_WEBHOOK_IMAGE wasn't found in the deployment of the member operator.")
+		err = initMember2Await.WaitUntilWebhookDeleted(t) // webhook on member2 should be deleted
+		require.NoError(t, err)
```

Comment:
Let's also verify that all there is only one `ValidatingWebhookConfiguration` and only one `MutatingWebhookConfiguration` CR created by any of the two member operators. We could use any of these two labels for that:
```
      app: member-operator-webhook
      toolchain.dev.openshift.com/provider: codeready-toolchain
```
In this way, you can verify that the ones managed by member-2 are deleted as well as the previous/old ones

Reply:
good point, I've added this check as part of https://github.com/codeready-toolchain/toolchain-e2e/pull/950/commits/9e8562ad953ad0a7571771ca473797e7ee713ce3 

Thanks!